backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5776630_sys_gh26293_BusinessPartnerAccountSheetReport_Process_Update.sql

```diff
@@ -0,0 +1,61 @@
+-- Run mode: SWING_CLIENT
+
+-- Process: BusinessPartnerAccountSheetReport(de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess)
+-- ParameterName: p_IsSOTrx
+-- 2025-11-13T13:47:06.953Z
+UPDATE AD_Process_Para SET AD_Reference_Value_ID=540528,Updated=TO_TIMESTAMP('2025-11-13 14:47:06.953','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Process_Para_ID=541756
+;
+
+-- Process: BusinessPartnerAccountSheetReport(de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess)
+-- ParameterName: p_IsSOTrx
+-- 2025-11-13T16:35:15.001Z
+UPDATE AD_Process_Para SET DefaultValue='Y',Updated=TO_TIMESTAMP('2025-11-13 17:35:15.001','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Process_Para_ID=541756
+;
+
+-- 2025-11-13T10:10:17.254Z
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,Description,EntityType,Help,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,584207,0,'p_IsSOTrx',TO_TIMESTAMP('2025-11-13 11:10:16.484','YYYY-MM-DD HH24:MI:SS.US'),100,'Legt fest, welche Transaktionsarten im Bericht berücksichtigt werden: * Ja → Nur Verkaufstransaktionen * Nein → Nur Einkaufstransaktionen * Leer → Verkaufs- und Einkaufstransaktionen','D','Legt fest, welche Transaktionsarten im Bericht berücksichtigt werden:
```

Comment:
why creating `p_IsSOTrx` element?

Reply:
to update ther description for this process only and not everry process